src/containers/ProjectContentsContainer.jsx

```diff
@@ -113,8 +113,15 @@ class ProjectContentsContainer extends Component {
           <ModalBody>
             <h2>Original</h2>
             <p>{this.state.fieldText}</p>
-            <h2>Translation</h2>
-            <input type="text" onChange={this.handleChange} autoFocus placeholder="Translate some text" />
+            <h2>Your translation</h2>
+            <MarkdownEditor
```

Comment:
This should be a textarea or text input if the field is plain text.

You should add a flag to fields that need the markdown editor, eg. `data-markdown=true` then check the flag here (`event.target.getAttribute('data-markdown')` and render either a plain text input or the markdown editor.